Version 0.1.0 not installing from tarball

```
  ----------------------------------------
  Failed building wheel for tskit
  Running setup.py clean for tskit
Failed to build tskit
Installing collected packages: tskit
  Running setup.py install for tskit ... error
    Complete output from command /usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ked0202h/tskit/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-zoniuy0o/install-record.txt --single-version-externally-managed --compile --user --prefix=:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/tskit
    copying tskit/drawing.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/exceptions.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/tables.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/formats.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/provenance.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/_version.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/trees.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/stats.py -> build/lib.linux-x86_64-3.6/tskit
    copying tskit/__init__.py -> build/lib.linux-x86_64-3.6/tskit
    running egg_info
    writing tskit.egg-info/PKG-INFO
    writing dependency_links to tskit.egg-info/dependency_links.txt
    writing entry points to tskit.egg-info/entry_points.txt
    writing requirements to tskit.egg-info/requires.txt
    writing top-level names to tskit.egg-info/top_level.txt
    reading manifest file 'tskit.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'tskit.egg-info/SOURCES.txt'
    running build_ext
    building '_tskit' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/lib
    creating build/temp.linux-x86_64-3.6/lib/kastore
    creating build/temp.linux-x86_64-3.6/lib/kastore/c
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -UNDEBUG -Ilib -Ilib/kastore/c -I/usr/include/python3.6m -I/home/jk/.local/lib/python3.6/site-packages/numpy/core/include -c _tskitmodule.c -o build/temp.linux-x86_64-3.6/_tskitmodule.o -std=c99
    _tskitmodule.c:29:10: fatal error: tskit.h: No such file or directory
     #include "tskit.h"
              ^~~~~~~~~
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -20,7 +20,8 @@ jobs:
             pip install --user -r python/requirements/development.txt
             pip install meson --user
             # tmp; until we have an msprime package install from here.
-            pip install --user --pre --upgrade tskit
+            # 0.1.0 is broken, so force 0.1.0a1 for now.
+            pip install --user --pre tskit==0.1.0a1
             pip install --user git+https://github.com/tskit-dev/msprime
             echo 'export PATH=/home/circleci/.local/bin:$PATH' >> $BASH_ENV
       - save_cache:
@@ -60,8 +61,7 @@ jobs:
             python setup.py build_ext
             python setup.py egg_info
             python setup.py bdist_wheel
-            # TODO This doesn't seem to be installing, fails on bdist_wheel build.
-            # pip install dist/*.tar.gz 
+            pip install dist/*.tar.gz 
       - run:
           name: Compile C
           command: |

--- a/python/MANIFEST.in
+++ b/python/MANIFEST.in
@@ -1,3 +1,4 @@
 include lib/kastore/c/kastore.h
+include lib/tsk*.h
 include LICENSE
 include tskit/_version.py

